donate.dd

```diff
@@ -0,0 +1,91 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(H4 The $(LINK2 $(ROOT_DIR)foundation.html, D Language Foundation) is an organization devoted to advancing
+     open source technology related to the D programming language.
+)
+
+The D Language foundation:
+
+$(COMMENT keep this list in sync with foundation.dd)
+$(UL
+    $(LI holds and defends the copyright and other intellectual property rights for the D programming language.)
+    $(LI provides resources for furthering the development and adoption of the D programming language.)
+    $(LI awards $(LINK2 http://dlang.org/blog/2016/12/05/the-d-language-foundations-scholarship-program, scholarships)
+        to highly skilled students.)
+    $(LI organizes the yearly $(HTTP dconf.org, D conference).)
```

Comment:
s/D conference/DConf, the international annual conference of D language developers and users/